pages/1.11/networking/DNS/index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: layout.pug
+navigationTitle:  DNS
+title: DNS Quick Reference
+menuWeight: 20
+excerpt:
+
+enterprise: false
+---
+
+<!-- This source repo for this topic is https://github.com/dcos/dcos-docs -->
+DC/OS provides a highly distributed and faulterant DNS-based service discovery mechanism. DNS is provided by two different components within DC/OS, [mesos-dns](/pages/1.11/networking/DNS/mesos-dns) and [dcos-dns](/pages/1.11/networking/DNS/dcos-dns). DNS service in DC/OS supports two  top-level domain names (TLD) `.mesos` and `.directory`. All records for the `.mesos` TLD are served by `mesos-dns` and all records for `.directory` TLD are served by `dcos-dns`. Please read the "Recommendation" section to better understand the usage of these two TLDs. Each of the TLD consist of multiple zones. Every DC/OS service gets multiple FQDN entries, from these different zones. Below we describe the different zones and the FQDN associated with those zones and the type of network connectivity that a service recieves when it is accessed using an FQDN from a particular zone.
```

Comment:
s/highly distributed and faulterant/distributed and fault tolerant/

Reply:
s/supports two  top-level/supports two top-level/

Reply:
s/top-level domain names (TLD) `.mesos` and `.directory`/top-level domain names (TLD): `.mesos` and `.directory`

Reply:
s/gets multiple FQDN entries, from these different zones/gets multiple FQDN entries from these different zones/